packages/contracts/contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -1049,25 +1073,42 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
      * Checks that the final batch context in a sequencer submission is valid.
```

Comment:
I think it's worth updating this comment while we're in here--maybe `Checks that the final batch context in a sequencer submission cannot have a higher timestamp/blocknumber than the first queue element in a future submission.`